### PR TITLE
Update functions in peopleservice.cpp; test ListAlluser() function; tests related to cin not solved yet.

### PR DESCRIPTION
**Pull request by Huazhu-W**

### CODE REVIEW

File: src/service/PeopleServices.cpp [hunk 1]
```diff
@@ -110,34 +106,46 @@ bool PeopleServices::initPassword() {
...
+                cout << "Hi," << People->getName() << "your account is locked , please contact your tutor or teacher for unlock" << endl;
+                lockUser(People->getUserId());
                 break;
             }
         }
     }
 
-    //TODO: need to replace by database
-    Storage::getCurrentUser()->setPassword(password);
+    People->setPassword(password);
```
Comment (AkideLiu) [inline src/service/PeopleServices.cpp:116]: need to call PeopleDao function  updatePeoplePassword()

File: src/service/PeopleServices.cpp [hunk 2]
```diff
@@ -110,34 +106,46 @@ bool PeopleServices::initPassword() {
...
 }
 
 bool PeopleServices::changePassword(int user_id, const string &oldPassword, const string &new_password) {
 
+    People* People = PeopleDao::selectOnePeople(user_id);
+
     if (oldPassword != new_password) {
-        //TODO: need to replace by database
-        Storage::getCurrentUser()->setPassword(new_password);
+        People->setPassword(new_password);
```
Comment (AkideLiu) [inline src/service/PeopleServices.cpp:126]: need to call PeopleDao function updatePeoplePassword()

**Review by AkideLiu — CHANGES_REQUESTED** [review]
need to call PeopleDao function updatePeoplePassword()

File: src/service/PeopleServices.cpp (hunk 2 above)
Comment (iamabin) [inline src/service/PeopleServices.cpp:126]: shoud like that
bool PeopleServices::changePassword(int user_id, const string &oldPassword, const string &new_password) {

    if (oldPassword != new_password) {
        //TODO: need to replace by database
        return PeopleDao::updatePeoplePassword(user_id, new_password);
       
    }
  
}

File: src/service/PeopleServices.cpp (hunk 1 above)
Comment (iamabin) [inline src/service/PeopleServices.cpp:116]: peopledao

File: src/service/PeopleServices.cpp [hunk 3]
```diff
@@ -110,34 +106,46 @@ bool PeopleServices::initPassword() {
                 cout << "password you entered is not the same, please try again ; you have " << count << "chance left ." << endl;
                 count--;
             } else {
-                //TODO: need to replace by database
-                cout << "Hi," << Storage::getCurrentUser()->getName() << "your account is locked , please contact your tutor or teacher for unlock" << endl;
-                lockUser(Storage::getCurrentUser()->getUserId());
+                cout << "Hi," << People->getName() << "your account is locked , please contact your tutor or teacher for unlock" << endl;
+                lockUser(People->getUserId());
```
Comment (iamabin) [inline src/service/PeopleServices.cpp:110]: peopledao

File: src/service/PeopleServices.cpp (hunk 2 above)
Comment (Huazhu-W) [reply src/service/PeopleServices.cpp]: can't get ur point

File: src/service/PeopleServices.cpp (hunk 1 above)
Comment (Huazhu-W) [reply src/service/PeopleServices.cpp]: Solved

File: src/service/PeopleServices.cpp (hunk 3 above)
Comment (Huazhu-W) [reply src/service/PeopleServices.cpp:110]: Solved

File: src/service/PeopleServices.cpp (hunk 2 above)
Comment (Huazhu-W) [reply src/service/PeopleServices.cpp]: Solved

File: src/service/PeopleServices.cpp (hunk 1 above)
Comment (Huazhu-W) [reply src/service/PeopleServices.cpp]: Solved

**Review by AkideLiu — APPROVED** [review]

**Pull request merged**